hypha/apply/home/migrations/0002_add_apply_homepage.py

```diff
@@ -34,6 +34,10 @@ def create_homepage(apps, schema_editor):
 
 class Migration(migrations.Migration):
 
+    run_before = [
+        ('wagtailcore', '0053_locale_model'),  # added for Wagtail 2.11 compatibility
+    ]
+
```

Comment:
This is required after wagtail 2.11.

see: https://docs.wagtail.io/en/stable/releases/2.11.html#run-before-declaration-needed-in-initial-homepage-migration

it was fixed for the [public site](https://github.com/HyphaApp/hypha/blob/9d7fdc3714ef3cea9e04800a6f52a2b35324f53f/hypha/public/home/migrations/0002_create_homepage.py#L40)..but also required for the apply site.